src/routes/index.js

```diff
@@ -1,12 +1,14 @@
 import express from 'express'
 import {getTestRoutes} from './testRoutes'
-
+import {getSummoner} from './matchHistoryRoutes'

```

Comment:
Instead of a function called getSummoner, let's do this instead:
```suggestion
import {getTFTRoutes} from './tftRoutes'

```